content/docs/buildpacks/nodejs.md

```diff
@@ -307,3 +307,9 @@ The Node.js CNB also supports simple apps that do not require third-party packag
 
 If no package manager is detected, the Node.js CNB will set the start command
 `node server.js`. The app name is ___not___ currently configurable.
+
+## Stack support
+For most apps, the Node.js Buildpack runs fine on the Base builder. But when the app
+requires compilation of native extensions using `node-gyp`, the buildpack requires that
+you use the Full builder. This is because `node-gyp` requires `python` that's absent on
+the Base builder, and the module may require other shared objects.
```

Comment:
```suggestion
the Base builder.
```

Reply:
I'm concerned not only about the requirements of `node-gyp` but about the requirements of the native module as such. Seems like base doesn't have many `lib*.so`s that full has.

Reply:
oh oops, I didn't mean to delete that. Looks good now